ecs-cli/modules/cli/servicediscovery/servicediscovery_helper.go

```diff
@@ -213,7 +218,28 @@ func getOutputIDFromStack(cfnClient cloudformation.CloudformationClient, stackNa
 }
 
 func cfnStackName(stackName, cluster, service string) string {
```

Comment:
can we rename the param `stackName` to `stackNameFmt` or something similar to be clear it's a format string (and not, say, something with more significance)?

Reply:
(also apologies for not noticing earlier)